app/controllers/reservations_controller.rb

```diff
@@ -4,14 +4,28 @@ class ReservationsController < ApplicationController
   before_action :set_reservation, only: %i[show edit update destroy]
   before_action :authenticate_user!
 
+  def find_reservations_by_user
+    @reservations = Reservation.where(user_id: 
+      User.find_by(email: params[:email]))
+    .joins(:seance).includes([:user, seance:[:movie]])
```

Comment:
```suggestion
    .includes([:user, seance:[:movie]])
```

Do you need `joins` here? :thinking:

Reply:
Without `joins` I'm not able to order records by `start_time`. It doesn't see this column

Reply:
valid point :+1: